experiments/snippets/autoLoadMore/autoLoadMore.js

```diff
@@ -0,0 +1,31 @@
+//inline javascript
```

Comment:
ESLint is pretty angry about the usage of spaces instead of tabs for this. 

If you run `npm run lint` from the repo's source directory, you'll see all the errors (assuming you've got node installed, run `npm ci`, etc. etc.)

To expedite this PR I'm just going to make Github suggestions on the lines.

Reply:
uploaded cleanup up version.  don't have node/don't have time, but used notepad++ to see the characters and changed them.